### PR TITLE
Remove preview script

### PR DESCRIPTION
**Pull request by carolynvs**
Now that you must preview the site through deconst client in order for the CSS to work right, I'm removing the preview script and updating the instructions.

### CODE REVIEW

**Comment by carolynvs** [comment]
@smashwilson Would you please you take a look at the updated instructions for previewing getcarina.com with Deconst Client?

File: README.md [hunk 1]
```diff
@@ -6,15 +6,25 @@
...
+1. Click the New button.
+1. Set the content repository to getcarina.com.
+1. Set the preparer to Jekyll.
+1. In the Content Repository Path section, click the browse button, then navigate to the location where you cloned this repository.
+1. In the Control Repository Location, click the browse button, then navigate to hte lcoation where you cloned the Nexus Control repository.
+1. Click the Create button.
+1. Wait until the deconst process is complete, then click the displayed link to preview the site.
 
-Jekyll is now serving this repo as a simple website, and will rebuild the site every time you save a file.
+    **Note:** If after 30 minutes you do not see a link and it is still in the preparing state, use [this workaround](https://github.com/deconst/client/issues/50) and add a comment that you have hit the bug as well.
```
Comment (smashwilson) [inline README.md]: 30 minutes? Oof. Let me see if I can time how long it takes me to render this repo...

**Comment by smashwilson** [comment]
:+1: Yep yep, this is all right. I do wish the client was a bit less unwieldy and more reliable...

File: README.md (hunk 1 above)
Comment (smashwilson) [inline README.md]: I just did a render that took 1 min 14 sec. I'd say give it maybe 5 minutes (just in case it needed to pull all of the containers)?

Comment (carolynvs) [inline README.md]: Yeah 30 minutes was pulled out of thin air. I wasn't sure if the first render was slower or not. I'll update it to 5 minutes.

**Comment by carolynvs** [comment]
The workaround I use is easy-peasy and I think it may be just me who has trouble with it. :smile: So no worries, it's a cool tool in my book.

**Pull request merged**